datafusion/sqllogictest/test_files/aggregate.slt

```diff
@@ -4536,19 +4536,14 @@ EXPLAIN SELECT DISTINCT c3 FROM aggregate_test_100 group by c3 limit 5;
 logical_plan
 01)Limit: skip=0, fetch=5
 02)--Aggregate: groupBy=[[aggregate_test_100.c3]], aggr=[[]]
-03)----Aggregate: groupBy=[[aggregate_test_100.c3]], aggr=[[]]
```

Comment:
These certainly look like bettr plans to me. 

However, I wonder if these changes now mean the query doesn't fulfill the original intent of the tests 

```sql
#
# Push limit into distinct group-by aggregation tests
#
```

Therefore I wonder if we should  add another test like

```
EXPLAIN SELECT DISTINCT c3, min(c1) FROM aggregate_test_100 group by c3 limit 5;
```

Reply:
Makes sense

Reply:
FWIW in my opinion it would be fine to add the other tests as a new PR too

Reply:
Sounds good, let's do it that way